src/tests/ll_verification.py

```diff
@@ -783,27 +941,9 @@ def do_ll_ddi_adv_bv_47_49_c(transport, upperTester, lowerTester, trace, packets
 
         if round.DataLength > 0:
             # Set adv data
-            maxFragmentSize = 251
             advData = random.randint(1, 255, round.DataLength)
-            remainingAdvData = advData[:]
-            firstFragment = True
-            while (len(remainingAdvData) > 0):
-                if firstFragment:
-                    if len(remainingAdvData) <= maxFragmentSize:
-                        op = FragmentOperation.COMPLETE_FRAGMENT
-                    else:
-                        op = FragmentOperation.FIRST_FRAGMENT
-                else:
-                    if len(remainingAdvData) <= maxFragmentSize:
-                        op = FragmentOperation.LAST_FRAGMENT
-                    else:
-                        op = FragmentOperation.INTERMEDIATE_FRAGMENT
-                endIndex = maxFragmentSize if len(remainingAdvData) >= maxFragmentSize else len(remainingAdvData)
-                status = le_set_extended_advertising_data(transport, upperTester, Handle, op, round.FragmentPref, remainingAdvData[:endIndex], 100)
-                if status != 0:
-                    return False
-                remainingAdvData = remainingAdvData[endIndex:]
-                firstFragment = False
+            if not set_complete_ext_adv_data(transport, upperTester, Handle, advData):
```

Comment:
I think the `round.fragmentPref` got lost in the refactoring

Reply:
Sure did, fixed now